apps/web/modules/auth/ui/screens/org-selection.screen.tsx

```
@@ -0,0 +1,13 @@
+import { OrganizationList } from "@clerk/nextjs";
+
+type Props = {};
+export const OrgSelectionScreen = ({}: Props) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)